dell-csi-helm-installer/csi-install.sh

```diff
@@ -43,7 +42,6 @@ function usage() {
   decho "  Optional"
   decho "  --release[=]<helm release>               Name to register with helm, default value will match the driver name"
   decho "  --upgrade                                Perform an upgrade of the specified driver, default is false"
-  decho "  --version                                Use this version for CSI Driver Image"
```

Comment:
why remove version? We need it to install custom images without --skip-verify

Reply:
This PR was created from the current state of the internal dell-csi-installer repo's master branch, so this line is being removed because it does not exist in that internal repo. In order to keep all of the driver repos in sync I've been told that we should keep all of the changes to these installer scripts in that internal repo and then propagate the changes to each of the driver's via a PR made by a utility script in that repo (that's how this PR was made).

If this is something that is needed, could you make a PR on the internal dell-csi-installer repo and then use the utility script to push the change here and to the other driver repos?